src/main/java/mate/jdbc/Main.java

```diff
@@ -1,7 +1,85 @@
+import mate.jdbc.service.CarService;
+import mate.jdbc.service.DriverService;
+import mate.jdbc.service.ManufacturerService;
+
 public class Main {
+    private static Injector injector = Injector.getInstance("mate.jdbc");
+
     public static void main(String[] args) {
-        // test your code here
+        CarService carService
+                = (CarService) injector.getInstance(CarService.class);
+        System.out.println("Removing cars from db ...");
+        carService.getAll().forEach(car -> carService.delete(car.getId()));
+        System.out.println("Creating drivers if they dont exist yet ...");
+        Driver olaf = new Driver("Olaf", "*GER*");
+        Driver gans = new Driver("Gans", "*KELLEN*");
+        List<Driver> driversFromGermany = new ArrayList<>();
+        driversFromGermany.add(olaf);
+        driversFromGermany.add(gans);
+        DriverService driverService
+                = (DriverService) injector.getInstance(DriverService.class);
+        driversFromGermany.forEach(driverService::create);
+        System.out.println("Printing all drivers in db ...");
+        driverService.getAll().forEach(driver -> System.out.println());
+        System.out.println("Creating new manufacturers ...");
+        Manufacturer manufacturerNato = new Manufacturer("NATO", "Germany");
+        ManufacturerService manufacturerService
+                = (ManufacturerService) injector.getInstance(ManufacturerService.class);
+        manufacturerService.create(manufacturerNato);
+        System.out.println("Creating car ...");
+        Car carOne = carService.create(new Car(
+                        "IRIS-T",
+                        manufacturerNato,
+                        driversFromGermany));
+        System.out.println("Printing car data from db " + carOne.getModel());
+        System.out.println(carService.get(carOne.getId()));
+        System.out.println("Updating car " + carOne.getModel());
+        carOne.setModel("Pajero");
+        Manufacturer manufacturerFromJapan
+                = new Manufacturer("Mitsubishi", "Japan");
+        manufacturerService.create(manufacturerFromJapan);
+        carOne.setManufacturer(manufacturerFromJapan);
+        System.out.println(carService.update(carOne));
+        System.out.println("Creating new driver ...");
+        Driver lii = new Driver("Lii", "*JAPAN*");
+        System.out.println(driverService.create(lii));
+        System.out.println("Updating driver for car " + carOne.getModel());
+        carOne.setDrivers(List.of(lii));
+        System.out.println(carService.update(carOne));
+        System.out.println("Printing car data from db ");
+        System.out.println(carService.get(carOne.getId()));
+        System.out.println("Deleting data from db car - " + carOne.getModel());
+        System.out.println(carService.delete(carOne.getId()));
+        System.out.println("Printing all cars by driver id ");
+        System.out.println(carService.getAllByDriver(1L));
```

Comment:
Do not hardcode ids! Let's take it from saved entity

Reply:
fixed